erts/emulator/test/port_SUITE_data/file_while_alive.c

```diff
@@ -0,0 +1,93 @@
+/*
+ * %CopyrightBegin%
+ *
+ * SPDX-FileCopyrightText: 2025 Ericsson and contributors
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * %CopyrightEnd%
+ */
+
+/*
+ * Test utility which waits for SIGTERM and responds by deleting a temp file
+ * path.
+ */
+#include <stdlib.h>
+#include <stdio.h>
+#include <signal.h>
+#include <errno.h>
+#include <fcntl.h>
+
+#ifndef __WIN32__
```

Comment:
Doesn't run on win32 yet (tests are not executed), but the headers at least allow for successful compilation.